.github/workflows/check-demo-notebooks.yml

```
@@ -0,0 +1,27 @@
+# This workflow will check if the run.py files in every demo match the run.ipynb notebooks.
+
+name: Check Demos Match Notebooks
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
```

I don't think we need labeled/unlabeled, but we might want reopened
```suggestion
    types: [opened, synchronize, reopened]
```

---

you're right we don't need labeled/unlabeled but do need reopened. committed